series/liberec-pyvo/events/2019-06-20-seste-zrusene.yaml

```
@@ -0,0 +1,14 @@
+city: liberec
+start: 2019-06-20 18:00:00
+name: Liberecké Pyvo
+number: 6
+topic: ZRUŠENÉ
+venue: depo
```

dá se zrušené pyvo dělat bez venue, nebo to neprojde validací?

---

Dobrá otázka. Ani mě to nenapadlo zkoušet, ale podle schématu by venue mělo být nepovinné. Jdu to zkusit.
```suggestion
- venue: depo
```

---

Tak to prošlo v #240. Takhle je to lepší, díky!